src/vs/workbench/contrib/preferences/browser/preferences.contribution.ts

```diff
@@ -552,6 +553,28 @@ class PreferencesActionsContribution extends Disposable implements IWorkbenchCon
 			}
 		});
 
+		registerAction2(class extends Action2 {
+			constructor() {
+				super({
+					id: SETTINGS_COMMAND_FILTER_TELEMETRY,
+					title: nls.localize({ key: 'miOpenTelemetrySettings', comment: ['&& denotes a mnemonic'] }, "&&Telemetry Settings"),
+					menu: {
+						id: MenuId.MenubarPreferencesMenu,
```

Comment:
I don't think we want this showing up in any menus we already have online services settings in the menus